parsing/unit_info.ml

```diff
@@ -22,13 +22,15 @@ type error = Invalid_encoding of string
 exception Error of error
 
 type t = {
-  source_file: filename;
+  original_source_file: filename;
+  raw_source_file: filename;
```

Comment:
The difference between `original` and `raw` feels a bit too subtle.  Maybe `human/generated`, `human/input`, or `original/input` to better indicate the causal relation between the two files?